### PR TITLE
Banner hotfix 

### PR DESCRIPTION
**Pull request by michalsidzej**
Old arrow looks bad on mobile 
image
So I fixed it 
Screenshot 2023-02-01 at 18 11 34

### CODE REVIEW

File: packages/frontend/src/components/navbar/Banner.tsx [hunk 1]
```diff
@@ -1,14 +1,17 @@
...
     <a
-      className="block text-center bg-pink-900 text-white text-xs font-medium px-4 py-1.5"
+      className="text-center bg-pink-900 text-white text-xs font-medium px-4 py-1.5 flex items-center justify-center"
       href="https://l2beat.notion.site/We-are-hiring-Work-at-L2BEAT-e4e637265ae94c5db7dfa2de336b940f"
       target="_blank"
     >
       We are hiring!
-      <span className="underline ml-2">Join L2BEAT &rarr;</span>
+      <span className="underline ml-2">Join L2BEAT</span>
+      <ArrowRightIcon className="fill-white w-3 h-3 ml-0.5" />
```
Comment (torztomasz) [inline packages/frontend/src/components/navbar/Banner.tsx:14]: why exactly do we need w-3 h-3?

**Review by torztomasz — APPROVED** [review]

File: packages/frontend/src/components/navbar/Banner.tsx (hunk 1 above)
Comment (michalsidzej) [reply packages/frontend/src/components/navbar/Banner.tsx:14]: Because otherwise the arrow is bigger

Comment (torztomasz) [reply packages/frontend/src/components/navbar/Banner.tsx:14]: So maybe we should constrain the size in SVG or className inside the ArrowRightIcon?

Comment (michalsidzej) [reply packages/frontend/src/components/navbar/Banner.tsx:14]: But we use this arrow in some other places

Comment (michalsidzej) [reply packages/frontend/src/components/navbar/Banner.tsx:14]: And it's bigger there

**Pull request merged**